papergres.go

```diff
@@ -533,22 +545,27 @@ func goToSQLName(name string) string {
 }
 
 // insertArgs creates the insert arg slice for an object
-func insertArgs(obj interface{}) []interface{} {
-	final, _ := prepareFields(obj)
+func insertArgs(obj interface{}, withPK bool) []interface{} {
```

Comment:
See the comment above regarding creating copies of the same variable when pushing functions on the stack.

Reply:
Value semantics are the idiomatic way to Go :)
Read the above comment for a detailed explanation on how the Go compiler works differently and uses escape analysis to make passing by value a much cheaper call.